libass/ass_render.c

```diff
@@ -2258,7 +2268,13 @@ static void render_and_combine_glyphs(ASS_Renderer *render_priv,
             }
             last_info = info;
 
-            if (!info->bm || !current_info) {
+            ASS_Vector pos, pos_o;
+            info->pos.x = double_to_d6(device_x + d6_to_double(info->pos.x) * render_priv->font_scale_x);
+            info->pos.y = double_to_d6(device_y) + info->pos.y;
+            get_bitmap_glyph(render_priv, info, &pos, &pos_o,
+                             &offset, !current_info->bitmap_count, flags);
+
+            if (!current_info || (!info->bm && !info->bm_o)) {
```

Comment:
Is the `!info->bm` to `!info->bm && !info->bm_o` change related to the rest of this commit?

Reply:
It's related to `pos` and `pos_o` handling above. Maybe it's indeed better to move that into one of previous commits.